### PR TITLE
EventGrid: Swagger for API Version 2019-06-01

### PR DESCRIPTION
**Pull request by ahamad-MS**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-06-01/EventGrid.json [hunk 1]
```diff
@@ -0,0 +1,3497 @@
...
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventGrid/topics": {
+      "get": {
+        "tags": [
+          "Topics"
+        ],
+        "summary": "List topics under a resource group.",
```
Comment (nschonni) [inline specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-06-01/EventGrid.json]:
```suggestion
      "description": "The number of results to return per page for the list operation. Valid range for top parameter is 1 to 100. If not specified, the default number of results to be returned is 20 items per page.",
```

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/6311 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/6311/commits/e6f855f464abdc5c0fde50b51f2abc3c12cc2fd0

File: specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-06-01/EventGrid.json (hunk 1 above)
Comment (ahamad-MS) [reply specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-06-01/EventGrid.json]: Thx. fixed

File: specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-06-01/EventGrid.json [hunk 2]
```diff
@@ -0,0 +1,3497 @@
...
+            "description": "EventSubscription CreateOrUpdate request accepted.",
+            "schema": {
+              "$ref": "#/definitions/EventSubscription"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
```
Comment (kalyanaj) [inline specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-06-01/EventGrid.json]: It looks like the earlier version had some more examples here, can you please double-check if they are still required?

Comment (ahamad-MS) [reply specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-06-01/EventGrid.json]: Thx. Fixed

**Comment by ahamad-MS** [comment]
@sarangan12 Can you please approve/merge the change if no more concerns?

**Review by sarangan12 — APPROVED** [review]

**Comment by sarangan12** [comment]
Adding ARM Review since this is a new version

**Comment by ahamad-MS** [comment]
@sarangan12  
We are not introducing any new ARM  changes. All the changes were already reviewed by ARM team in person and were part of previous preview Swagger.

**Comment by ravbhatnagar** [comment]
This is same as 2019-02-01-preview version. New resource type domains is being added. This has already been reviewed and approved by ARM for the preview. Signing off from ARM side.

**Pull request merged**